AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/InitialSync/InitialSyncOperationTests.swift

```diff
@@ -48,11 +51,33 @@ class InitialSyncOperationTests: XCTestCase {
             api: apiPlugin,
             reconciliationQueue: reconciliationQueue,
             storageAdapter: storageAdapter,
-            dataStoreConfiguration: .default) { _ in }
+            dataStoreConfiguration: .default)
+
+        let syncStartedReceived = expectation(description: "Sync started received, sync operation started")
+        let syncCompletionReceived = expectation(description: "Sync completion received, sync operation is complete")
+        let finishedReceived = expectation(description: "InitialSyncOperation finishe offering items")
+        let sink = operation
+        .publisher
+        .sink(receiveCompletion: { _ in
+            syncCompletionReceived.fulfill()
+        }, receiveValue: { value in
+            switch value {
+            case .started(modelType: let modelType, syncType: let syncType):
+                XCTAssertEqual(modelType.modelName, "MockSynced")
+                XCTAssertEqual(syncType, SyncType.fullSync)
```

Comment:
nit: does this work?
```suggestion
                XCTAssertEqual(syncType, .fullSync)
```

Reply:
Ya, it works. I removed SyncType across the places